pkgs/development/interpreters/octave/default.nix

```diff
@@ -119,6 +119,10 @@ let
       sha256 = "sha256-1KnYHz9ntKbgfLeoDcsQrV6RdvzDB2LHCoFYCmS4sLY=";
     };
 
+    patches = [
+      ./patches/bug62436.patch
```

Comment:
```suggestion
      # https://savannah.gnu.org/support/download.php?file_id=53190
      ./patches/bug62436.patch
```

Reply:
Oh I'm sorry @KarlJoad :grin: but I meant to comment this link:

https://savannah.gnu.org/bugs/?func=detailitem&item_id=62436